api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -275,12 +275,9 @@ private User initializeUserIfNeeded() {
     user.setGivenName(request.getGivenName());
     userDao.save(user);
 
-    try {
-      return ResponseEntity.status(HttpStatus.CREATED).body(profileService.getProfile(user));
-    } catch (ApiException e) {
-      log.log(Level.SEVERE, "Error getting user profile: {0}".format(e.getResponseBody()), e);
-      return ResponseEntity.status(e.getCode()).build();
-    }
+    // TODO(dmohs): This should be 201 Created with no body, but the UI's swagger-generated code
+    // doesn't allow this. Fix.
```

Comment:
Does this mean that the UI is going to barf if we return nothing here?

This is why @blrubenstein introduced the EmptyResponse thing:

https://github.com/all-of-us/workbench/blob/da87e79111a0fbcfd941d0402fcbec8b2ca215c7/api/src/main/java/org/pmiops/workbench/api/ClusterController.java#L102

Which you could use here if you had to. 

But if we have to return something, I'd just as soon return the thing we just created rather than nothing.

Reply:
It handles a 204 with no body correctly, but I believe a 201 with no body is allowed by the HTTP spec, so not handling that correctly is a bug.